Documentation/Running.rst

```diff
@@ -154,7 +154,7 @@ Running unit and functional tests
 .. important::
 
     Before running any commands you need to install all dependencies with
-    `runTests.sh -s composer update`
+    `./Build/Scripts/runTests.sh -s composer update`
```

Comment:
We should also use `.. code-block:: bash` here (like with the other commands in this file).